test/database_functions.go

```diff
@@ -0,0 +1,127 @@
+package test
+
+import (
+  "database/sql"
+  "fmt"
+  "testing"
+
+  // Microsoft SQL Database driver
+  _ "github.com/denisenkom/go-mssqldb"
+)
+
```

Comment:
We may consider to add this functionality to Terratest to benefit more people.

Reply:
This is the official recommended one: https://github.com/golang/go/wiki/SQLDrivers